debugfs.c

```diff
@@ -363,11 +363,16 @@ static ssize_t mwl_debugfs_info_read(struct file *file, char __user *ubuf,
 		len += scnprintf(p + len, size - len,
 			 "-----------------------=>  address| address|qlen|fw_desc_cnt\n");
 		spin_lock_irqsave(&pcie_priv->tx_desc_lock, flags);
-		len += scnprintf(p + len, size - len,
-				"wcb_base0   : %x => %8x|%8p|%4d|%d\n", get_hw_spec->wcb_base0, *((unsigned int *)le32_to_cpu(get_hw_spec->wcb_base0)),(void *)*((unsigned int *)le32_to_cpu(get_hw_spec->wcb_base0)),skb_queue_len(&pcie_priv->txq[0]),pcie_priv->fw_desc_cnt[0]);
+		len += scnprintf(p + len, size - len, "wcb_base0   : %x => %8x|%8p|%4u|%d\n",
+				 get_hw_spec->wcb_base0, le32_to_cpu(get_hw_spec->wcb_base0),
```

Comment:
Maybe you were just trying to fix the "int to pointer cast" warning, but you also changed how this function works.

Reply:
Basically. Any suggestions?